chaosaws/elbv2/actions.py

```diff
@@ -32,9 +32,157 @@ def deregister_target(tg_name: str,
     return client.deregister_targets(TargetGroupArn=tg_arn[tg_name],
                                      Targets=[{'Id': random_target}])
 
+
+def set_security_groups(load_balancer_names: List[str] = None,
+                        security_group_ids: List[str] = None,
+                        configuration: Configuration = None,
+                        secrets: Secrets = None) -> List[AWSResponse]:
+    """
+    Changes the security groups for the specified load balancer(s).
+    This action will replace the existing security groups on an application
+    load balancer with the specified security groups.
+
+    Parameters:
+        - load_balancer_names: a list of load balancer names
+        - security_group_ids: a list of security group ids
+
+    returns:
+        [
+            {
+                'LoadBalancerArn': 'string',
+                'SecurityGroupIds': ['sg-0000000', 'sg-0000001']
+            },
+            ...
+        ]
+    """
+    if not load_balancer_names:
```

Comment:
This is not necessary. If you don't set a default value for a parameter and this parameter is not providedi n the experiment, Chaos Toolkit will fail with a validation error.

Reply:
I agree, this check is just unnecessary code. Will correct.